### ISSUE TITLE
Unintuitive output for vsk.arc's start and stop params

### ISSUE THREAD

**Issue by bcongdon** [abey79/vsketch#96]
I'm a bit confused by the "start" and "stop" parameters of `vsk.arc`. They don't seem to do what I'd expect:

```py
self.ellipseMode("radius")
self.arc(0, 0, w=5, h=5, start=0, stop=90, degrees=True)
self.arc(15, 0, w=5, h=5, start=90, stop=180, degrees=True)
self.arc(30, 0, w=5, h=5, start=180, stop=270, degrees=True)
self.arc(45, 0, w=5, h=5, start=270, stop=360, degrees=True)
self.circle(60, 0, diameter=5)
```
![Screen Shot 2021-03-11 at 7 21 54 PM](https://user-images.githubusercontent.com/706257/110887393-180f5100-829f-11eb-8f8c-70df144ebcea.png)

With the above code, I'd expect the output to look more like this:

![Screen Shot 2021-03-11 at 7 22 06 PM](https://user-images.githubusercontent.com/706257/110887405-1c3b6e80-829f-11eb-9ee5-f0e7204f9e76.png)

Put another way, it looks like the "start" and "stop" degrees are not always relative to the same absolute starting reference point. This same behavior seems to happen in radians mode too. Am I misunderstanding the intended usage?

**Comment by abey79** [comment]
vsketch uses the [unit circle](https://en.wikipedia.org/wiki/Unit_circle) convention: zero angle points to the right and positive angles extend counter-clockwise. In this regards, the output of your code is as expected. You were instead expecting the "compass" conversion (0 angle pointing top, positive angles extending clockwise).

I will clarify the documentation on this.

**Comment by bcongdon** [comment]
Ah, I see. That makes a lot more sense now, thanks for the clarification!

**Comment by abey79** [comment]
No worries.

I'm going to reopen this issue so I don't forget to update `arc`'s documentation.

**Comment by lenary** [comment]
This caught me out too, because the `+y` direction of the unit circle for `arc` and the `+y` direction for coordinates do not match, if you use `vsk.size(…, landscape=True, center=False)`. In this case, Increasing `y` takes you "down" the page in coordinate space, whereas for your unit circle increasing `y` takes you "up" the page.

**Comment by abey79** [comment]
I agree this is not ideal. I should add this to the list of breaking changes to make pre-1.0.
